BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -267,3 +266,37 @@ fileprivate final class ScrollerWrapperView: UIView {
     }
 
 }
+
+private extension ScrollerWrapperView {
+
+    final class CustomScrollView: UIScrollView {
+
+        override var refreshControl: UIRefreshControl? {
+            get {
+                if #available(iOS 10.0, *) {
+                    return super.refreshControl
+                } else {
+                    return refreshControl
```

Comment:
I guess this isn't an recursive bomb since Swift knows to return the underlying value and not go through the getter it's already in?

Reply:
No, I think it's a 💣.

Reply:
Yea it is, I realized this after I blew up my own feature.

Using a `private var _refreshControl` to manage this for < 10. Not sure if I should still set it on >= 10 though.

Reply:
Leaving it nil is ok, you can save a `objc_retain` call and make everything 0.00000000000001% faster.